src/java/org/apache/cassandra/config/Config.java

```diff
@@ -79,6 +79,7 @@ public static Set<String> splitCommaDelimited(String src)
     public String authenticator;
     public String authorizer;
     public String role_manager;
+    public String crypto_provider;
```

Comment:
This should be of type `ParameterizedClass`. That way it will be configured same way as e.g. `seed_provider`.

Reply:
What's the benefit of making it` ParameterizedClass`? We don't need any other input other than the provider name.

Reply:
The benefit is that it is configurable. Are you 100% sure that there will never be a provider which is configurable? Your provider is not, are you sure you can say it above any possible provider yet to be implemented? 

Also, it is very easy to do that in terms of how it looks like in cassandra.yaml. Instead of this

    `crypto_provider: this.is.my.ClassImpl`

It would be like

    crypto_provider:
      - class_name: this.is.my.ClassImpl

I do not think it is a lot to ask given the fact the we open ourselves to a lot of possibilities when it is potentially configurable.